test/specs/utils/helpers.js

```diff
@@ -561,6 +562,7 @@ describe('utils helpers', () => {
 		);
 	});
 	describe('#shouldUseJSONOutput', () => {
+		beforeEach(setUpUtilHelpers);
```

Comment:
Prefer doing this at the outermost level, or having individual set up functions.